bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -507,12 +507,17 @@ func (s *ksServer) getRegistryUri(registry *RegistryConfig) (string, error) {
 }
 
 func runCmd(rawcmd string) error {
-	cmd := exec.Command("sh", "-c", rawcmd)
-	err := cmd.Run()
-	if err != nil {
-		log.Errorf("Error occrued during execute cmd %v. Error: %v", rawcmd, err)
+	for retry := 0; ; retry++ {
```

Comment:
Should we use a library like: https://github.com/cenkalti/backoff to handle retries?

Reply:
Sounds good